meinberlin/assets/scss/utility.scss

```diff
@@ -48,7 +48,7 @@
     margin: 0;
     padding: 0;
 
-    > li {
+    > * {
```

Comment:
now it targets children but won't overwrite additional styles as it's not a high order selector, checked comp lib and it seems ok